### PR TITLE
CCD-2420: Probate PROD - Allow ad-hoc data extract requests to complete

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-2420

### Change description ###

Changed to support probate batch rerun

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/ccd/ccd-data-store-api/prod.yaml [hunk 1]
```diff
@@ -29,5 +29,5 @@ spec:
         IDAM_OIDC_URL: https://hmcts-access.service.gov.uk
         LOGGING_LEVEL_UK_GOV_HMCTS_CCD_SECURITY_FILTERS: DEBUG
         REQUEST_SCOPE_CACHED_CASE_TYPES: GrantOfRepresentation
-        REQUEST_SCOPE_CACHED_CASE_TYPES_FROM_HOUR: 1
+        REQUEST_SCOPE_CACHED_CASE_TYPES_FROM_HOUR: 20
```
Comment (MSancaktutar) [inline k8s/namespaces/ccd/ccd-data-store-api/prod.yaml:32]: this doesn't work, unfortunately.
The hour limits are applied within the same day.
I suggest something like 20-24. And then, the revert will bring it back to 1-6.
I can merge the revert PR just before midnight.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
needs correction

File: k8s/namespaces/ccd/ccd-data-store-api/prod.yaml (hunk 1 above)
Comment (rajaram-nagarajan) [reply k8s/namespaces/ccd/ccd-data-store-api/prod.yaml:32]: You are right, Mutlu. Just checked the condition in the data-store. In that case, yes, we can go for 20-24.

**Review by MSancaktutar — DISMISSED** [review]
lgtm

**Review by MSancaktutar — APPROVED** [review]
lgtm

**Pull request merged**